client/v2/client.go

```diff
@@ -330,7 +330,7 @@ func (bp *batchpoints) SetRetentionPolicy(rp string) {
 
 // Point represents a single data point
 type Point struct {
-	pt models.Point
```

Comment:
This was me. I needed this so I could easily create `client.Point` from an existing `models.Point`. The current `NewPoint` function relies on all the separate aspects of a point, which seemed clunky for the way I wanted to use the client.